### PR TITLE
replace instanceof in brand checks with isPrototypeOf

### PR DESCRIPTION
**Pull request by KhafraDev**
As a side note, but not majorly important: isPrototypeOf is also resilient against Symbol.hasInstance shenanigans. The real benefit is needing less imports to perform brand checks now that it is unified.

### CODE REVIEW

File: lib/web/cookies/index.js [hunk 1]
```diff
@@ -5,9 +5,7 @@ const { stringify } = require('./util')
 const { webidl } = require('../fetch/webidl')
 const { Headers } = require('../fetch/headers')
 
-const headersToBrandCheck = globalThis.Headers
-  ? [Headers, globalThis.Headers]
-  : [Headers]
+const brandChecks = webidl.brandCheckMultiple([Headers, globalThis.Headers].filter(Boolean))
```
Comment (Uzlopak) [inline lib/web/cookies/index.js:8]: What if you run node with --no-experimental-fetch? Then globalThis.Headers will be undefined. Thus breaking the code.

**Review by Uzlopak — CHANGES_REQUESTED** [review]
See comment

File: lib/web/cookies/index.js (hunk 1 above)
Comment (KhafraDev) [reply lib/web/cookies/index.js:8]: that's what the .filter(Boolean) is for

**Review by mcollina — APPROVED** [review]
lgtm

**Review by metcoder95 — APPROVED** [review]

File: lib/web/cookies/index.js (hunk 1 above)
Comment (Uzlopak) [reply lib/web/cookies/index.js:8]: Ah right

**Comment by KhafraDev** [comment]
@mcollina I don't think this is semver-major?

**Pull request merged**